common/services/allocation.js

```diff
@@ -79,8 +80,9 @@ const allocationService = {
 
       return value
     })
-  },
-  transform({ includeCancelled = false } = {}) {
+  }
+
+  static transform({ includeCancelled = false } = {}) {
```

Comment:
Static so it can be used outside of an instance (it's used in the getAll function above). Doesn't use the req context in any way, so being static works.